test/extended/util/client.go

```diff
@@ -113,8 +113,11 @@ var (
 
 // loadConfig gets the User organization workspace and home workspace servers
 func loadConfig() {
-	var err error
-	// If not set "E2E_TEST_CONTEXT" use "kcp-stable" testContext by default
+	var (
+		rootServer, orgServer string
```

Comment:
Isn't `orgServer` a global variable defined in line 110? I think you might need to remove this local variable.

Reply:
Also, maybe consider `rootServer` to be global variable too. Might be useful in other places.

Reply:
Good catch! I miss it. Fixed.